src/c2s/mongoose_c2s.erl

```diff
@@ -15,7 +15,8 @@
 -export([start_link/2]).
 -export([get_host_type/1, get_lserver/1, get_sid/1, get_jid/1,
          get_mod_state/2, remove_mod_state/2,
-         get_ip/1, get_socket/1, get_lang/1, get_stream_id/1]).
+         get_ip/1, get_socket/1, get_lang/1, get_stream_id/1,
+         exit/2]).
 -export([filter_mechanism/2, c2s_stream_error/2, maybe_retry_state/1,
          reroute/2, stop/2, merge_states/2]).
```

Comment:
Just for clarity, I'd export `stop/2` and `exit/2` together, and likewise I'd put their implementations together below (currently `exit/2` is at the end of the file simply). A matter of style, makes it more visible in the far future that there's two very similar functions.